recipe/post-link.bat

```diff
@@ -1,4 +1,4 @@
 @echo off
-
-"%PREFIX%\Scripts\jupyter.exe" nbextensions_configurator enable --sys-prefix
-if errorlevel 1 exit 1
```

Comment:
Well, there's no roll-backs. So if the thing doesn't install (or uninstall), it holds up a whole environment creation? Tough call.